--- a/main/MainApp.java
+++ b/main/MainApp.java
@@ -12,6 +12,7 @@
 import java.util.List;
 import mainGuis.EquilinoxGuis;
 import mainGuis.GuiRepository;
+import nl.kingdev.eml.EML;
 import shops.ShopItem;
 import toolbox.MyKeyboard;
 import toolbox.OpenGlError;
@@ -19,10 +20,11 @@
 public class MainApp {
    public static final boolean CHEATS = false;
    private static final String VER = "0";
-   public static final String VERSION = "Equilinox_0";
-   public static final String VERSION_STRING = "Version 1.4.0c";
+   public static final String VERSION = "Equilinox_0-EML";
+   public static final String VERSION_STRING = "Version 1.4.0c-EML";
 
    public static void main(String[] args) {
+      EML.emlMain();
       ErrorLogger.init();
       EngineMaster.init(Camera.getCamera());
       FirstScreenUi screen = new FirstScreenUi();
