--- a/main/MainApp.java
+++ b/main/MainApp.java
@@ -14,6 +14,7 @@
 import java.util.List;
 import mainGuis.EquilinoxGuis;
 import mainGuis.GuiRepository;
+import nl.kingdev.eml.EML;
 import shops.ShopItem;
 import toolbox.MyKeyboard;
 import toolbox.OpenGlError;
@@ -25,6 +26,8 @@
    public static final String VERSION_STRING = "Version 1.5.0";
 
    public static void main(String[] args) {
+      //EML: Init
+      EML.emlMain();
       ErrorManager.init(new File("EquilinoxErrorLogs"), "An error has caused the program to crash, sorry for the inconvenience! Please email the dev at thinmatrix@gmail.com and copy-paste the error message below:");
       EngineMaster.init(Camera.getCamera());
       FirstScreenUi screen = new FirstScreenUi();
